### DISCUSSION TITLE
Add a link to the setup guide on your blog

### DISCUSSION THREAD

**Discussion by doej1367** [Skylar-Tech/node-red-contrib-matrix-chat#83] [Ideas]
When I was first reading the [README.md](https://github.com/Skylar-Tech/node-red-contrib-matrix-chat#end-to-end-encryption-notes) file, it was unclear to me how to set up the bot account. Maybe it would be a good idea to include a link to the blog post [ [https://skylar.tech/matrix-chat-bot-module-for-node-red/](https://skylar.tech/matrix-chat-bot-module-for-node-red/) ] covering that topic, e.g. in the `End-to-End Encryption Notes` section of the README file.

**Comment by skylord123** [comment]
@doej1367 thanks for the input! This would be a great change. I will get this added to my todo list.

**Comment by skylord123** [comment]
@doej1367 I added this and it will be in the next release. Again, thanks for pointing this out.
